Makefile

```diff
@@ -1,6 +1,9 @@
-.PHONY: image
+.PHONY: image test
 
 IMAGE_NAME ?= codeclimate/codeclimate-eslint
 
 image:
 	docker build --rm -t $(IMAGE_NAME) .
+
+test: image
+	docker run $(IMAGE_NAME) node_modules/.bin/mocha
```

Comment:
To be a bit more conventional, can you specify a task inside the `package.json` file? Something like:

```json
"scripts": {
    "test": "mocha test"
  }
```

And then this invocation can be `docker run $(IMAGE_NAME) npm run test`